Utils/JSONSchema/infer.h

```diff
@@ -160,6 +160,59 @@ CURRENT_STRUCT(String) {
   // LCOV_EXCL_STOP
 };
 
+CURRENT_STRUCT(Integer) {
+  CURRENT_FIELD(sum, int64_t, 0);           // Assume the sum would fit an int64. Because why not? -- D.K.
```

Comment:
Well, fuck.

It's intentional. The schema will be inferred correctly, just the mean/stddev would fail.
I thought of min/max, but even that requires separate signed/unsigned handling, so I disregarded the option for now.

Well, fuck.